### PR TITLE
Update login screen dismissal

### LINKED ISSUE #229: Login screen at start-up should emphasize the "no thanks" and de-emphasize the "log in"
The current login at start-up screen is confusing for new users.

### PR DESCRIPTION
**Pull request by JulianGro**
This PR changes the option to dismiss login, to make it a lot more visible.
1Vircadia_699
Closes #229

Also changed the domain-server favicon while I was at it. Seemed to small of a change to go through the trouble of making another PR for it.

### CODE REVIEW

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-404ffec_7b34872b.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-404ffec.tar_7b34872b.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-404ffec_7b34872b.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-404ffec.tar_7b34872b.gz

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-404ffec_7b34872b.exe

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-8841d5c_56362461.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-8841d5c.tar_56362461.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-8841d5c.tar_56362461.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-8841d5c_56362461.sh

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-8841d5c_56362461.exe

File: interface/resources/qml/LoginDialog/LinkAccountBody.qml [hunk 1]
```diff
@@ -485,7 +486,10 @@ Item {
                     }
                 }
             }
+            
+
```
Comment (ctrlaltdavid) [inline interface/resources/qml/LoginDialog/LinkAccountBody.qml:490]: Unnecessary whitespace?

File: interface/resources/qml/LoginDialog/LinkAccountBody.qml [hunk 2]
```diff
@@ -542,38 +546,55 @@ Item {
...
+                font.family: linkAccountBody.fontFamily
+                font.pixelSize: linkAccountBody.textFieldFontSize
+                font.bold: linkAccountBody.fontBold
+                verticalAlignment: Text.AlignVCenter
+                horizontalAlignment: Text.AlignHCenter
+                visible: loginDialog.getLoginDialogPoppedUp() && !linkAccountBody.linkSteam && !linkAccountBody.linkOculus;
             }
+
+            TextMetrics {
+				id: dismissButtonTextMetrics
```
Comment (ctrlaltdavid) [inline interface/resources/qml/LoginDialog/LinkAccountBody.qml:568]: Tabs should be 4 spaces. (Makes it hard to see what's changed when there are tabs.)

File: interface/resources/qml/LoginDialog/LoggingInBody.qml [hunk 3]
```diff
@@ -3,6 +3,7 @@
 //
 //  Created by Wayne Chen on 10/18/18
 //  Copyright 2018 High Fidelity, Inc.
+//  Copyright 2020 Vircadia contributors.
```
Comment (ctrlaltdavid) [inline interface/resources/qml/LoginDialog/LoggingInBody.qml:6]: No Vircadia changes in this file? If so then don't need to add this copyright.

File: interface/resources/qml/LoginDialog/LinkAccountBody.qml [hunk 4]
```diff
@@ -542,37 +544,54 @@ Item {
...
+                fontBold: linkAccountBody.fontBold
+                visible: loginDialog.getLoginDialogPoppedUp() && !linkAccountBody.linkSteam && !linkAccountBody.linkOculus;
+                onClicked: {
+                    if (linkAccountBody.loginDialogPoppedUp) {
+                        var data = {
+                          "action": "user dismissed login screen"
+                     };
+                     UserActivityLogger.logAction("encourageLoginDialog", data);
+                       loginDialog.dismissLoginDialog();
+                       }
```
Comment (ctrlaltdavid) [inline interface/resources/qml/LoginDialog/LinkAccountBody.qml:592]: Indentation of this JavaScript code needs fixing.

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-580d7b9.tar_3fe1ddde.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-580d7b9_3fe1ddde.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-580d7b9.tar_3fe1ddde.gz

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-580d7b9_3fe1ddde.exe

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-457df91_5b0855fa.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-457df91_5b0855fa.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-457df91.tar_5b0855fa.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-457df91.tar_5b0855fa.gz

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR471/Vircadia-Alpha-PR471-457df91_5b0855fa.exe
